RCode/NEFI/run_bayes_model.R

```diff
@@ -19,16 +20,20 @@ source('RCode/helper_functions/plug_n_play_functions.R')
 start_date = '2007-01-01' # in YYYY-MM-DD format; 1st 
 end_date = '2015-12-31' # Excluding 2016-2017 to use as sample data
 site = c('midge') # options are midge, coffin, newbury, or fichter 
-model_timestep = 1 # model timestep in days if filling in dates
-fill_dates = FALSE  # T/F for filling in dates w/o observations with NA's 
+model_timestep = 7 # model timestep in days if filling in dates
+fill_dates = TRUE  # T/F for filling in dates w/o observations with NA's 
 
-model_name = 'RandomWalk' # options are RandomWalk, RandomWalkZip, Logistic, Exponential, DayLength, DayLength_Quad, RandomYear, TempExp, Temp_Quad,  ChangepointTempExp
+model_name = 'Exponential_RandomYear' # options are RandomWalk, RandomWalkZip, Logistic, Exponential, DayLength, DayLength_Quad, RandomYear, TempExp, Temp_Quad,  ChangepointTempExp
 model=paste0("RCode/NEFI/Jags_Models/",model_name, '.R') #Do not edit
 
 #How many times do you want to sample to get predictive interval for each sampling day?
 #Edit nsamp to reflect a subset of total number of samples
 nsamp = 500 
 
+#My local directory - use as a temporary file repository for plot files before uploading
+#to Google Drive for the team to see :)
+my_directory <- "C:/Users/Mary Lofton/Desktop/MEL_Bayes"
```

Comment:
We could also have a team temporary directory that doesn't get `git committed` but _is_ uploaded to google drive. Then people wouldn't have to change this line to be their own temp directory. 
I've used the file path `Figs/NEFI/tmp/` and there is a gitignore for `tmp/*` already so it will not `git commit` these